unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/DomainProfileTopInfoCell/DomainProfileTopInfoCell.swift

```diff
@@ -135,7 +135,8 @@ extension DomainProfileTopInfoCell {
                 case .image(let image, _):
                     imageView.image = image
                 case .imageURL(let url, _):
-                    let image = await appContext.imageLoadingService.loadImage(from: .url(url),
+                    let image = await appContext.imageLoadingService.loadImage(from: .url(url,
```

Comment:
5 or more calls with a new parameter and the same value...

wouldn't be nicer to make `Constants.downloadedImageMaxSize` a default value for `maxSize`?

Reply:
@rommex They're similar but not the same, `downloadedImageMaxSize` and `downloadedIconMaxSize`